aptos-move/aptos-vm/src/aptos_vm_impl.rs

```diff
@@ -42,6 +42,8 @@ use move_vm_runtime::logging::expect_no_verification_errors;
 use move_vm_types::gas::UnmeteredGasMeter;
 use std::sync::Arc;
 
+pub const MAXIMUM_APPROVED_TRANSACTION_SIZE: u64 = 1024 * 1024;
```

Comment:
Why limit approved governance proposal txs?

Reply:
because if it is too big, someone can flood the network with valid transactions... eventually they'll succeed, but why allow someone to push out 8 MB transactions? Everything should be limited.